### PR TITLE
Update customize page

### LINKED ISSUE #107: “Popular subgames” and “popular mods” is outdated and unfair
These sections are pretty bad.

Especially because they become outdated so quickly.

Only 4 subgames + 1 fake subgame are showcased, the selection seems arbitrary. Tutorial isn't even a game. Dreambuilder is not a subgame, but a mod.
Subgames should really be selected by quality, not (perceived) popularity. Popularity is a terrible measurement.

Mods are even worse. This section is horribly outdated and will probably always be. Many mods are so old, they really shouldn't be used anymore.

There is no easy fix. Just trying to keep the sections updated by hand via PRs is a bad solution as PRs are usually extremely time-consuming, and we shouldn't waste so much time whenever a high-quality mod pops up.
I don't have a good idea what to do with mods, if you have one, please let me know!

For subgames, using PRs *might* *kinda* work as there are only a very few subgames of decent quality. Still, it's not really a long-term solution.

But the inclusion criteria for subgames must change. Popularity is completely irrelevant if you only play in singleplayer. What matters is the quality.

I propose to select subgames which are:
* Showing Minetest in a good light
* Enjoyable
* Stable
* Polished (to some degree)
* Working in latest stable Minetest
* Completed, or if WIP, the WIP state is already so good and stable you can play it normally already without having the impression of something lacking
* Passing the Six Hour Test (it must not run out of new content to explore before 6 hours of playing) (non-sandbox games are excempt)
* Having a crafting guide (unless they don't need one)

Also, the subgames/mods page should be re-evaluated after each MT release, in order to scrap everything which has ceased to function.

### PR DESCRIPTION
**Pull request by rubenwardy**
[screenshot](https://i.rubenwardy.com/mHEL6Ro91g.png)
  
web link with no CSS: https://rubenwardy.com/minetest.github.io/customize/

fixes #107

### CODE REVIEW

File: customize.html [hunk 1]
```diff
@@ -11,56 +11,50 @@
 
 <h1 id="subgames">Subgames</h1>
 <p>
-  <a href="http://wiki.minetest.net/Subgames">Subgames</a> form a foundation for the game.  Different subgames have different objectives – such as survival, building, or player vs. player.
-  Usually Minetest comes with Minetest Game, which supplies the default items.
+  <a href="http://wiki.minetest.net/Subgames">Subgames</a> provide a foundation for
+  the game. Different subgames have different objectives – such as survival,
```
Comment (sfan5) [inline customize.html]: don't you need to "escape" this unicode char in HTML?

File: customize.html [hunk 2]
```diff
@@ -11,56 +11,50 @@
...
+  <div class="col-sm-4">
     <h3>Building</h3>
     <ul>
       <li><a href="https://forum.minetest.net/viewtopic.php?t=2041">Home Decor</a></li>
       <li><a href="https://forum.minetest.net/viewtopic.php?t=12534">X-Decor</a></li>
       <li><a href="https://forum.minetest.net/viewtopic.php?t=509">More Blocks</a></li>
       <li><a href="https://forum.minetest.net/viewtopic.php?t=5682">Quartz</a></li>
       <li><a href="https://forum.minetest.net/viewtopic.php?t=2984">Streets</a></li>
-      <li><a href="https://forum.minetest.net/viewtopic.php?t=15651">Colored Clay</a></li>
-      <li><a href="https://forum.minetest.net/viewtopic.php?t=638">Nuke</a></li>
```
Comment (sfan5) [inline customize.html]: hmm who put "nuke" under "Building"?
must've been me...

File: customize.html [hunk 3]
```diff
@@ -69,84 +63,81 @@ <h3>Map generation</h3>
...
 
-  <div class="col-sm-6">
-    <h3>Plants &amp; Mobs</h3>
+<div class="row">
+  <div class="col-sm-4">
+    <h3>Plants, Animals &amp; Monsters</h3>
     <ul>
       <li><a href="https://forum.minetest.net/viewtopic.php?t=9019">Farming Redo</a></li>
       <li><a href="https://forum.minetest.net/viewtopic.php?t=629">Animals Modpack</a></li>
-      <li><a href="https://forum.minetest.net/viewtopic.php?t=8638">Creatures Mob Engine</a></li>
```
Comment (sfan5) [inline customize.html]: perhaps suggest [this](https://github.com/minetest-mods/mob-engine) instead?
Though it doesn't seem to have a forum topic

File: customize.html [hunk 4]
```diff
@@ -169,7 +160,6 @@ <h3>Popular Texture Packs</h3>
       <li><a href="https://forum.minetest.net/viewtopic.php?t=1583">HDX Textures</a></li>
       <li><a href="https://forum.minetest.net/viewtopic.php?t=14132">PixelBOX Reloaded</a></li>
       <li><a href="https://forum.minetest.net/viewtopic.php?t=14289">PixelPerfection</a></li>
-      <li><a href="https://forum.minetest.net/viewtopic.php?t=9314">Summerfields</a></li>
```
Comment (sfan5) [inline customize.html]: why remove this one? it still looks usable

File: customize.html (hunk 1 above)
Comment (Calinou) [reply customize.html]: No, not in HTML5. Unicode characters can be used directly just fine, only `<`, `>` and `&` need to be escaped.

File: customize.html (hunk 4 above)
Comment (rubenwardy) [reply customize.html]: is has an unfree license

File: customize.html (hunk 3 above)
Comment (rubenwardy) [reply customize.html]: For consistency, I've disregarded multiple forks which do not have forum topics. I think it's best just for the author to make a topic for them

**Pull request merged**